helper/storagepacker/storagepacker.go

```diff
@@ -128,34 +106,25 @@ func (s *Bucket) upsert(item *Item) error {
 	return nil
 }
 
-// BucketIndex returns the bucket key index for a given storage key
-func (s *StoragePacker) BucketIndex(key string) uint8 {
-	hf := md5.New()
-	hf.Write([]byte(key))
-	return uint8(hf.Sum(nil)[0])
-}
-
-// BucketKey returns the bucket key for a given item ID
+// BucketKey returns the storage key of the bucket where the given item will be
+// stored.
 func (s *StoragePacker) BucketKey(itemID string) string {
-	return strconv.Itoa(int(s.BucketIndex(itemID)))
+	hf := md5.New()
+	hf.Write([]byte(itemID))
+	index := uint8(hf.Sum(nil)[0])
+	return s.viewPrefix + strconv.Itoa(int(index))
 }
 
-// DeleteItem removes the storage entry which the given key refers to from its
-// corresponding bucket.
+// DeleteItem removes the item from the respective bucket
 func (s *StoragePacker) DeleteItem(itemID string) error {
-
 	if itemID == "" {
 		return fmt.Errorf("empty item ID")
 	}
 
-	// Get the bucket key
 	bucketKey := s.BucketKey(itemID)
 
-	// Prepend the view prefix
-	bucketPath := s.BucketPath(bucketKey)
```

Comment:
Was this prepending the viewPrefix path twice, once in here and then again `s.view.Get`?